stable/distributed-tensorflow/Chart.yaml

```
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: A Helm chart for running distributed TensorFlow on Kubernetes
 name: distributed-tensorflow
-version: 0.1.3
-appVersion: 1.6.0
+version: 0.1.4
```

The change presented in this PR changes `apiVersion: apps/v1beta2` to `apiVersion: apps/v1` - I believe this is major change, so it should bump Chart major version (and not just patch).

```suggestion
version: 0.2.0
```

---

Done